app/assets/javascripts/discourse/app/components/user-menu/items-list.hbs

```diff
@@ -14,14 +14,14 @@
   </ul>
   <div class="panel-body-bottom">
     {{#if this.showAllHref}}
-      <DButton
-        class="show-all"
-        @action={{this.showAll}}
-        @translatedAriaLabel={{this.showAllTitle}}
-        @translatedTitle={{this.showAllTitle}}
+      <a
+        class="btn btn-default show-all"
```

Comment:
BTW, if you pass `@href=` to `<DButton` it'll automatically render as an `<a`

But this is also fine, especially since we weren't using any of DButton built-in translation stuff anyway.

Reply:
TIL about `@href` turning the `DButton` into a `<a>`.